diff --git a/unhaggle/nvd/migrations/0001_initial.py b/unhaggle/nvd/migrations/0001_initial.py
index c3b1fcf3c5..e4dcc1c181 100644
--- a/unhaggle/nvd/migrations/0001_initial.py
+++ b/unhaggle/nvd/migrations/0001_initial.py
@@ -271,7 +271,7 @@ class Migration(migrations.Migration):
                 ('last_updated', models.DateTimeField(auto_now=True, db_index=True, null=True)),
                 ('content_en', models.CharField(max_length=512)),
                 ('content_fr', models.CharField(max_length=512)),
-                ('schema_category', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='features', to='nvd.JatoSchemaCategory', to_field='schema_id')),
+                # ('schema_category', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='features', to='nvd.JatoSchemaCategory', to_field='schema_id')),
                 ('vehicle', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='nvd.JatoVehicle', to_field='vehicle_id')),
             ],
             bases=(models.Model, uh_core.models.cache.GetCachedMixin, unhaggle.nvd.models.JatoCommon),
@@ -285,6 +285,13 @@ class Migration(migrations.Migration):
             name='schema_id',
             field=models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, related_name='category', to='nvd.JatoSchemaInfo'),
         ),
+        # NOTE: Inserted after schema_id addition above
+        migrations.AddField(
+            model_name='jatovehiclestandardfeature',
+            name='schema_category',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE,
+                                    related_name='features', to='nvd.JatoSchemaCategory', to_field='schema_id'),
+        ),
         migrations.AddField(
             model_name='jatooptiontext',
             name='vehicle',
